dataprofiler/profilers/utils.py

```diff
@@ -124,7 +122,9 @@ def shuffle_in_chunks(
 
         # Generate random list of indexes
         lower_bound_list = np.array(range(j, j + true_chunk_size))
-        random_list = rng.integers(lower_bound_list, data_length)
+        random_list = np.empty(len(lower_bound_list))
+        for i in range(len(lower_bound_list)):
+            random_list[i] = rng.randint(lower_bound_list[i], data_length)
```

Comment:
Is this change needed? Or why was it needed? I'm concerned that looping as well is going to be expensive.